### DISCUSSION TITLE
Group Tabs

### DISCUSSION THREAD

**Discussion by BraveBrowserFan** [zen-browser/desktop#3917] [Ideas]
Hi

I like to see on Zen Browser to have group tabs like this

![image](https://github.com/user-attachments/assets/72141041-3149-4fb7-84cc-fcff300d1cf3) on brave browser. thanks

**Comment by 12th-devs** [comment]
tab groups are possible, you can enable them in about:config but mozzila is still working on them and they are not fully functional, you will also want custom CSS to make it look good.

**Comment by mr-cheffy** [comment]
This has been recuested in many posts now... Please avoid duplicating
